application.py

```diff
@@ -6,7 +6,8 @@
 
 application = create_app(os.getenv('DM_ENVIRONMENT') or 'development')
 manager = Manager(application)
-manager.add_command("runserver", Server(port=5002))
+manager.add_command("runserver", Server(
+    os.getenv('DM_HOST') or '127.0.0.1', port=5002))
```

Comment:
I don't know - it just seemed like specifically setting the address the server runs on might have an impact.  If AWS needs it to run on something other than `127.0.0.1` then it would need to be set as DM_HOST in the Cloudformation templates.  But as far as I know this might just work.  @allait probably knows more than me about this to say.  Or we could just suck it and see.